code/modules/shuttle/shuttle.dm

```diff
@@ -343,6 +343,9 @@
 	var/rechargeTime = 0 //time spent after arrival before being able to launch again
 	var/prearrivalTime = 0 //delay after call time finishes for sound effects, explosions, etc.
 
+	var/playing_Launch_Announcement_Alarm = 0 // 0 = off ; 1 = on
+	var/datum/looping_sound/looping_launch_announcement_alarm/alarm_sound_loop
```

Comment:
qdel_null this on destroy()

Reply:
In shuttle.dm...
is this correct?
![image](https://github.com/cmss13-devs/cmss13/assets/142365554/e5f4bbdd-9805-4fe7-8b74-b8502d09c523)